shopinvader_product_price_tax/models/shopinvader_variant.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2021 Camptocamp (http://www.camptocamp.com).
+# @author Iván Todorovich <ivan.todorovich@gmail.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class ShopinvaderVariant(models.Model):
+    _inherit = "shopinvader.variant"
+
+    def _get_price(self, qty=1.0, pricelist=None, fposition=None, company=None):
+        res = super()._get_price(
+            qty=qty, pricelist=pricelist, fposition=fposition, company=company
+        )
+        # Apply company
+        product = self.record_id
```

Comment:
can't we have an helper method like `_get_price_variables` to get: product, company, taxes, and pricelist?

Not a blocker of course!